packages/@angular/cli/blueprints/ng/files/__path__/main.ts

```diff
@@ -8,4 +8,6 @@ if (environment.production) {
   enableProdMode();
 }
 
-platformBrowserDynamic().bootstrapModule(AppModule);
+platformBrowserDynamic().bootstrapModule(AppModule)
+  .then(success => console.log('Bootstrap success'))
```

Comment:
I'd remove the success block; as it is most likely going to be deleted immediately and mainly there for illustrative purposes.